podcasts/Onboarding/Plus/UpgradeLandingView.swift

```diff
@@ -277,13 +320,15 @@ struct UpgradeCard: View {
                                 .fixedSize(horizontal: false, vertical: true)
                         }
                     }
-                }
-                .padding(.bottom, 24)
 
-                Button(tier.buttonLabel) {
-                    viewModel.unlockTapped(plan: tier.plan, selectedPrice: currentPrice.wrappedValue)
+                    Group {
+                        termsAndConditions
+                    }
+                    .font(style: .footnote).fixedSize(horizontal: false, vertical: true)
+                    .tint(.black)
+                    .opacity(0.64)
```

Comment:
No need for the Group here. 

```suggestion
                    termsAndConditions
                    .font(style: .footnote).fixedSize(horizontal: false, vertical: true)
                    .tint(.black)
                    .opacity(0.64)
```

Reply:
Ops, left it by mistake, done in a1c6d072c374a2a84c66fa733c843db5cabd8b69